editor/js/libs/ui.js

```diff
@@ -756,6 +780,18 @@ UI.Integer = function ( number ) {
 	var pointer = [ 0, 0 ];
 	var prevPointer = [ 0, 0 ];
 
+	dom.addEventListener( 'keydown', function ( event ) {
+
+		event.stopPropagation();
+
+		if ( event.keyCode === 38 || event.keyCode === 40 ) {
+			scope.nudgeNumericValue( event.keyCode, event.shiftKey );
+			event.preventDefault();
```

Comment:
This preventDefault prevents the cursor from moving around, which is what happens naturally when you use the up and down keys and an input is focused.